app/models/hud_reports/generator_base.rb

```diff
@@ -42,16 +42,16 @@ def self.report_year_slug
     def queue
       @report.state = 'Waiting'
       @report.question_names = self.class.questions.keys
-      @report.save
+      @report.save!
       Reporting::Hud::RunReportJob.perform_later(self.class.name, @report.id)
     end
 
     def run!(email: true, manual: true)
       @report.state = 'Waiting'
       @report.question_names = self.class.questions.keys
       @report.manual = manual
-      @report.save
-      Reporting::Hud::RunReportJob.perform_now(self.class.name, @report.id, email: email)
+      @report.save!
+      Reporting::Hud::RunReportJob.new.perform(self.class.name, @report.id, email: email)
```

Comment:
This makes me slightly nervous.  If memory serves `perform_now` and `new.perform` run in different contexts (ActiveJob vs. DelayedJob, I think).  What's the reasoning behind this change?

Reply:
I was having trouble with the job swallowing the stack trace, not quite sure why. I'll change it back. The difference is that invoking it this way should avoid ActiveJob & DelayedJob entirely

Reply:
That's totally fine!  Since we were running them `now` anyway, I think it makes more sense to do what you have.  Thanks!